torch/custom_class.h

```diff
@@ -402,6 +402,23 @@ class class_ : public ::torch::detail::class_base {
     registerCustomClassMethod(std::move(method));
     return method_val;
   }
+
+  // Wrapper function to force method deregistration on shutdown.
+  static void registerCustomClassMethod(std::unique_ptr<jit::Function> method) {
```

Comment:
Why couldn't we reuse RAII mechanism, already implemented in https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/dispatch/RegistrationHandleRAII.h

Reply:
Just letting you know, I'm not the original author of this code, so feel free to edit as necessary.